src/main/resources/reference.conf

```diff
@@ -408,6 +408,9 @@ resourceTypes = {
       create_study = {
         description = "Has permission to create study"
       }
+      share_policy::steward = {
```

Comment:
This line is causing an error when I try to run this branch of SAM locally.

```
Exception in thread "main" com.typesafe.config.ConfigException$Parse: reference.conf @ file:/app/target/scala-2.12/classes/reference.conf: 411: Expecting a value but got wrong token: ':' (if you intended ':' to be part of a key or string value, try enclosing the key or value in double quotes, or you may be able to rename the file .properties rather than .conf)
```

Can you wrap this in double quotes?